test_values.go

```diff
@@ -17,4 +19,16 @@ var testGateway = New(
 	os.Getenv("BRAINTREE_PRIV_KEY"),
 )
 
+var testTimeZone = func() *time.Location {
+	tzName := os.Getenv("BRAINTREE_TIMEZONE")
+	if tzName == "" {
+		tzName = "US/Mountain"
```

Comment:
Agreed, but the account we test with on travis is setup with this timezone, and according to the Braintree docs it can't be changed without emailing Braintree Support.